sdk/eventhub/Azure.Messaging.EventHubs.Processor/src/EventProcessorClient.cs

```diff
@@ -952,7 +952,7 @@ private async Task RunAsync(CancellationToken cancellationToken)
                     }
                     catch (EventHubsException ex)
                     {
-                        var errorEventArgs = new ProcessErrorEventArgs(null, ex.Message, ex, cancellationToken);
+                        var errorEventArgs = new ProcessErrorEventArgs(null, ex.Message, ex.InnerException ?? ex, cancellationToken);
```

Comment:
Should we not us the same exception for the message?

Reply:
This is the exception thrown by the `PartitionLoadBalancer` class. We talked about it while reviewing Chris' PR. Basically, the load balancer wraps the actual exception in an `EventHubsException` before throwing it to the processor. We are using the inner exception here to remove the unnecessary wrapper around it.

Reply:
ooohhhh.... right.   My bad.  Forgot about that.